Add aarch64 python 3.10 build support

yasdum opened the pull request:
Add aarch64 python 3.10 build support

manoj7410 committed:
Merge pull request #53 from google-coral/FixInstallRequirements

Adding code to download testdata for movenet_pose_estimation

dmitriykovalev committed:
Add Apple M1 build support

DavidFV2296190 committed:
Antialias renamed to Lanczos (#130)

* Update detect_image.py

https://pillow.readthedocs.io/en/stable/releasenotes/2.7.0.html#antialias-renamed-to-lanczos

* antialias-renamed-to-lanczos

Namburger commented on examples/classify_image.py:
```diff
@@ -76,7 +76,7 @@ def main():
 
   size = common.input_size(interpreter)
-  image = Image.open(args.input).convert('RGB').resize(size, Image.ANTIALIAS)
+  image = Image.open(args.input).convert('RGB').resize(size, Image.LANCZOS)
```
There're too many things going on in this one PR, changes like this should be in a separate cl.

Namburger commented on Makefile:
```diff
@@ -16,21 +16,28 @@ PYTHON ?= $(shell which python3)
+CPU ?= darwin_arm64
+DARWIN_CPU := darwin_arm64
+else
 CPU ?= darwin
```
Thanks @yasdum!
I'm not super familiar with MacOS, why is there a need for a `darwin` and a `darwin_x86_64`?

scott306lr committed:
A better way to ensure pipeline closed successfully. (#80)

usmank13 committed:
Fix an IndexError edge case (#84)

There was an index error that occurs sometimes when running a custom edge tpu model. Disagreement between count and len(scores). Confirmed that this resolves the bug
